src/nvc.c

```diff
@@ -120,13 +138,16 @@ static int analyse(int argc, char **argv)
       { "psl",             no_argument,       0, 'P' },
       { "relax",           required_argument, 0, 'X' },
       { "relaxed",         no_argument,       0, 'R' },
+      { "define",          required_argument, 0, 'd' },
```

Comment:
```suggestion
      { "define",          required_argument, 0, 'D' },
```

Reply:
Not really, I think the `val` from `options` struct should be unique, and `D` is there already for `--dump-llvm`.